api/HTMLElement.json

```diff
@@ -868,6 +868,67 @@
           }
         }
       },
+      "enterKeyHint": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "77"
```

Comment:
Version 77 confirmed by https://www.chromestatus.com/feature/5654529808793600

Reply:
Also confirmed by:
- testing https://mdn-bcd-collector.appspot.com/tests/api/HTMLElement/enterKeyHint in Chrome 76 and 77
- https://storage.googleapis.com/chromium-find-releases-static/ae7.html#ae770f5e4a155964130ac56abff9438e488aca50